manifests/g/Google/IAPDesktop/2.18.631/Google.IAPDesktop.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate
+
+PackageIdentifier: Google.IAPDesktop
+PackageVersion: 2.18.631
+PackageLocale: en-US
+Publisher: Google

```

Comment:
```suggestion
Publisher: Google Inc

```

Reply:
Oh my bad, I was just following the previous layout. I was going to put "Google Inc" but I saw that "Google" was used previously.